pkg/sinker/types.go

```diff
@@ -8,6 +8,8 @@ type Response struct {
 	Success bool `json:"success"`
 	// Err represents the error message when "success" is false.
 	Err string `json:"err,omitempty"`
+	// Fallback is true if the message to be sent to the fallback sink.
+	Fallback bool `json:"fallback,omitempty"`
```

Comment:
Shall we combine `Fallback` and `Success` to use a new property?

Reply:
I thought of using an enum, but enums might get tricky in some languages so went with two separate boolean values.